core/lib/spree/core/mail_method.rb

```diff
@@ -6,7 +6,7 @@ def initialize(options={})
 
       def deliver_now!(mail)
         if Config.enable_mail_delivery
-          mailer.deliver!(mail)
```

Comment:
any reason you dropped the `!` off the end?

Reply:
Yeah, I was looking at the docs and it said this: 

![image](https://user-images.githubusercontent.com/7515374/35284613-3a587b84-0029-11e8-8471-a427bbe583de.png)

so I went with `deliver_now` instead. But I can always change it to `deliver_now!` if we don't care too much.

Reply:
I think we should keep the same functionality that exists before unless we understand why it was a bang method before and decide to change the behaviour :) 
Otherwise lgtm

Reply:
Makes sense! I'll change that then 👍